Rubberduck.Parsing/ComReflection/ReferencedDeclarationsCollector.cs

```diff
@@ -142,49 +141,47 @@ public List<Declaration> LoadDeclarationsFromLibrary()
             {
                 var moduleName = new QualifiedModuleName(_referenceName, _path,
                     module.Type == DeclarationType.Enumeration || module.Type == DeclarationType.UserDefinedType
-                        ? string.Format("_{0}", module.Name)
+                        ? $"_{module.Name}"
                         : module.Name);
 
                 var declaration = CreateModuleDeclaration(module, moduleName, project, GetModuleAttributes(module));
                 var moduleTree = new SerializableDeclarationTree(declaration);
                 _declarations.Add(declaration);
                 _serialized.AddDeclaration(moduleTree);
 
-                var membered = module as IComTypeWithMembers;
-                if (membered != null)
+                switch (module)
                 {
-                    CreateMemberDeclarations(membered.Members, moduleName, declaration, moduleTree, membered.DefaultMember);
-                    var coClass = membered as ComCoClass;
-                    if (coClass != null)
+                    case IComTypeWithMembers membered:
+                        CreateMemberDeclarations(membered.Members, moduleName, declaration, moduleTree, membered.DefaultMember);
+                        if (membered is ComCoClass coClass)
+                        {
+                            CreateMemberDeclarations(coClass.SourceMembers, moduleName, declaration, moduleTree, coClass.DefaultMember, true);
+                        }
+                        break;
+                    case ComEnumeration enumeration:
                     {
```

Comment:
I attempted to remove them and was getting an error. I believe it's because of the `members` variable. Without the `{ ... }` block it comes up with a naming clash.